source/CIS/cis-2.rst

```diff
@@ -688,10 +688,8 @@ Smart contract limitations
 
 A number of limitations are important to be aware of:
 
-- Smart contract function parameters are limited to 1 KiB.
+- Smart contract function parameters are limited to 65 KiB.
```

Comment:
It is `64KiB` Or technically 65535 bytes at most. I thinkg the latter should be stated since `64KiB` is ambiguous (is it 65536 allowed, or not?)